canopy/app/src/components/navigation/SideNav.js

```diff
@@ -28,17 +30,36 @@ function SideNav(props) {
       return <NavItem key={path} path={path} label={displayName} logo={logo} />;
     }
   );
+
   return (
     <div className="side-nav">
       <a href="/" className="brand">
         <h5>Canopy</h5>
       </a>
       <hr />
       <div className="nav-items">{userSaplingTabs}</div>
+      <hr className="bottom" />
+      <ProfileTab />
```

Comment:
In a future PR, we should have a way to add this tab iff the profile sapling is installed.